### DISCUSSION TITLE
linkerd2-proxy in docker compose

### DISCUSSION THREAD

**Discussion by Dkhusainov** [linkerd/linkerd2#10448] [Help]
Hello. I'm currently using linkerd1 in a docker compose with a simple consul namer and path identifiers. Can I use linkerd2-proxy as a drop-in-replacement? Can't find any info on how to use it outside of kubernetes.

**Comment by jeremychase** [comment]
Neither Linkerd2 nor linkerd2-proxy is a drop-in replacement for Linkerd1. As of 2.12, Linkerd2 is Kubernetes native and requires a cluster to run.

It is possible to run a lightweight Kubernetes cluster locally using [kind](https://github.com/kubernetes-sigs/kind/) or [k3d](https://github.com/k3d-io/k3d), and you could [convert your docker-compose.yaml](https://kubernetes.io/docs/tasks/configure-pod-container/translate-compose-kubernetes/) into Kubernetes manifests.

**Comment by Dkhusainov** [comment]
Thanks for the help!
